includes/Pages/PageMain.php

```diff
@@ -68,22 +68,29 @@ protected function main()
 
     /**
      * @param PdoDatabase $database
+     * @param SiteConfiguration $config
      * @param bool        $seeAllRequests
      *
      * @internal param User $currentUser
      */
-    private function setupLastFiveClosedData(PdoDatabase $database, $seeAllRequests)
+    private function setupLastFiveClosedData(PdoDatabase $database, SiteConfiguration $config, $seeAllRequests)
     {
         $this->assign('showLastFive', $seeAllRequests);
         if (!$seeAllRequests) {
             return;
         }
 
+        $queryExcludeDropped = "";
+        if($config->getEmailConfirmationEnabled()) {
```

Comment:
Code style nit on spacing:
```suggestion
        if ($config->getEmailConfirmationEnabled()) {
```